Lib/asyncio/queues.py

```diff
@@ -17,7 +17,7 @@ class QueueFull(Exception):
     pass
 
 
-class Queue(mixins._LoopBoundedMixin):
+class Queue(mixins._LoopBoundMixin):
```

Comment:
Why `loop=_marker` trick is not used here?

Reply:
My bad, fixed